gnosis/cowswap
# Feature/1484 ios bg load delay

# Summary

Fixes #1484 

on ios devices there were a delay before bg is loaded and shown to user

  # To Test

1. Open any page
2. try to switch between light and dark mode. delay should appear
